recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: nibabel
+  version: "2.0.2"
+
+source:
+  fn: nibabel-2.0.2.tar.gz
+  url: https://pypi.python.org/packages/source/n/nibabel/nibabel-2.0.2.tar.gz
+  md5: fa89163149e35f52e237b2cb56b1c606
+requirements:
```

Comment:
Above `requirements` add a build section like [this]( https://github.com/conda-forge/django-extensions-feedstock/blob/master/recipe/meta.yaml#L10-L12 ). Also, delete `build.sh`, but keep `bld.bat`.